slick/src/main/scala/slick/jdbc/MySQLProfile.scala

```diff
@@ -65,6 +65,7 @@ trait MySQLProfile extends JdbcProfile { profile =>
     - SqlCapabilities.sequenceLimited
     - RelationalCapabilities.joinFull
     - JdbcCapabilities.nullableNoDefault
+    - JdbcCapabilities.distinguishesIntTypes
```

Comment:
How about you add this as a comment above distinguishesIntTypes, so we don't forget, with a link back to this PR?